README.md

```diff
@@ -50,7 +50,7 @@ constructor.
 | `includePath` | `true` |  whether or not to include normalized URL path as a metric label - see [about `includePath`](#about-includepath) below |
 | `normalizePath` | |  a `function(req)` - generates path values from the express `req` object |
 | `paramIgnores` | `[]` |  array of path parameters _not_ to replace. _Use with caution as this may cause high label cardinality._ |
-| `formatStatusCode` | `(res) => res.status_code || res.statusCode` |  a `function(res)` - generates path values from the express `res` object |
+| `formatStatusCode` | `(res) => res.status_code` || res.statusCode` |  a `function(res)` - generates path values from the express `res` object |
```

Comment:
actually this _was_ correct... The second cell supposed to read `(res) => res.status_code || res.statusCode`

Reply:
The problem is that `||` is being interpreted incorrectly... Maybe it should be `\|\|`?